jest/identity.js

```diff
@@ -0,0 +1,29 @@
+/* eslint-disable no-var, comma-dangle, no-unused-vars, global-require */
+let Reflect;
```

Comment:
Maybe we should add a description explaining what this file does?

Reply:
I removed one dependency `identity-obj-proxy` and copy the code there, yarn2 did not like it and was breaking all test. https://github.com/verdaccio/ui/issues/93 meant to verify whether still need this, so I might left a comment on the ticket so anyone can match this change.